web-ng/src/app/components/map/map.component.css

```diff
@@ -13,4 +13,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
\ No newline at end of file
+
+.reposition-confirm-dialog {
+  position: absolute;
+  bottom: 2em;
+  left: 50%;
+  transform: translate(-50%, 0);
+  width: 407px;
+  height: 66px;
+  border-radius: 8px;
+  background: #188038;
```

Comment:
Can we use style vars and theme for colors in this sheet? See other scss for examples

Reply:
It seems this color is specific to this confirm dialog, extracted as a variable in the scss file.